src/app/modules/account/pages/registration/components/user-registration/user-registration.component.ts

```diff
@@ -28,9 +28,44 @@ import {Account} from "../../../../../../models/account.model";
   templateUrl: "./user-registration.component.html",
   styleUrls: ["./user-registration.component.scss"],
   standalone: true,
-  imports: [IonicModule, CommonModule, FormsModule],
+  imports: [IonicModule, CommonModule, FormsModule,ReactiveFormsModule],
 })
-export class UserRegistrationComponent {
+export class UserRegistrationComponent implements OnInit{
   @Input() account?: Partial<Account>;
-  constructor() {}
+  registrationForm: FormGroup;
+
+  constructor(private fb: FormBuilder) {
+    this.registrationForm = this.fb.group({
```

Comment:
Can you include `type='user'` here or on submit.  We'll need the type when we're saving this data.